pkg/pci/pci_linux.go

```diff
@@ -40,7 +40,7 @@ func getDeviceModaliasPath(ctx *context.Context, address string) string {
 	}
 	return filepath.Join(
 		paths.SysBusPciDevices,
-		pciAddr.Domain+":"+pciAddr.Bus+":"+pciAddr.Slot+"."+pciAddr.Function,
+		pciAddr.Domain+":"+pciAddr.Bus+":"+pciAddr.Device+"."+pciAddr.Function,
```

Comment:
We should just use `pciAddr.String()` here - perhaps in a later patch

Reply:
++ fixed.